### PR TITLE
Transmute 112

### PR DESCRIPTION
**Pull request by jimmyhmiller**
I want to transmute this rule in order to get rid of the win con. This game is about the experiment not winning.

### CODE REVIEW

**Comment by ArchmageInc** [comment]
yay

File: laws.md [hunk 1]
```diff
@@ -86,3 +84,6 @@ This rule takes precedence over every other rule determining the winner.
 
 **301.** Ammendment of rule **201.**: Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
 
+
+**302** The state of affairs that constitutes winning may not be altered from achieving n points to any other state of affairs. The magnitude of n and the means of earning points may be changed, and rules that establish a winner when play cannot continue may be enacted and (while they are mutable) be amended or repealed.
```
Comment (ArchmageInc) [inline laws.md:88]: Not that it is critical, but should we indicate that this is a Transmutation of rule **112.**?

Comment (jimmyhmiller) [inline laws.md:88]: The commit shows that. In the file itself there is no 112. So it doesn't make sense to me to do that. Also technically that would be changing the rule and transmuting it.

**Comment by dilbertKocik** [comment]
I approve this proposal.

File: laws.md (hunk 1 above)
Comment (ArchmageInc) [inline laws.md:88]: I guess. I just see it easier when the affected rule is in the rule, rather than have to see the commit history. But I get your point.

**Comment by jimmyhmiller** [comment]
How I read this rule is that I need to explicitly end debate and open up voting. So I don't think Brandon's vote would count yet because I haven't opened it up for voting. I know here it is just a transmutation, but I think this is important for later rules changes.

> **111.** If a rule-change as proposed is unclear, ambiguous, paradoxical, or destructive of play, or if it arguably consists of two or more rule-changes compounded or is an amendment that makes no difference, or if it is otherwise of questionable value, then the other players may suggest amendments or argue against the proposal before the vote. A reasonable time must be allowed for this debate. The proponent decides the final form in which the proposal is to be voted on and, unless the Judge has been asked to do so, also decides the time to end debate and vote.

**Comment by ArchmageInc** [comment]
Okay, well when you do open it to voting, I vote yay. ;)

**Comment by jimmyhmiller** [comment]
Voting is now open.

**Comment by ArchmageInc** [comment]
yay

**Comment by dilbertKocik** [comment]
yay again?

**Comment by jimmyhmiller** [comment]
yay

**Pull request merged**